source/templates.tex

```diff
@@ -1856,22 +1855,22 @@
 template<typename T, template<typename> concept CC>
 concept D = CC<T>;
 
-template<typename T,
+template<typename U,
          template<typename> concept CT,
          template<typename, template<typename> concept> concept CU>
-int f() requires CU<T, CT>;
-int _ = f<int, C, D>();
-\end{codeblock}
-In this example, the associated constraint of \tcode{f}
-is a concept-dependent constraint $CI$
-whose expression is the concept-id \tcode{CU<T, CT>} with the mapping
-$\tcode{T} \mapsto \tcode{T}, \tcode{CT} \mapsto \tcode{CT}, \tcode{CU} \mapsto \tcode{CU}$.\\
-$CI'$ is the result of substituting \tcode{D} into $CI$.\\
-We consider the normal form $CI''$ of \tcode{D<T, CT>},
+int f() requires CU<U, CT>;
+int i = f<int, C, D>();
+\end{codeblock}
+In this example, the associated constraints of \tcode{f}
+are a concept-dependent constraint
+whose expression is the concept-id \tcode{CU<U, CT>} with the mapping
+$\tcode{U} \mapsto \tcode{U}, \tcode{CT} \mapsto \tcode{CT}, \tcode{CU} \mapsto \tcode{CU}$.\\
```

Comment:
Thanks, this is nice. Could we please also remove those manual linebreaks as much as possible, @lprv?